packages/perseus/src/widgets/interactive-graphs/backgrounds/axis-labels.test.ts

```diff
@@ -94,3 +131,34 @@ describe("getLabelPosition", () => {
         expect(getLabelPosition(graphInfo, labelLocation)).toEqual(expected);
     });
 });
+
+describe("clampLabelPosition", () => {
+    it("should clamp the label position down to the max values", () => {
+        const graphInfo: GraphDimensions = {
+            range: [
+                [-10, 10],
+                [-10, 10],
+            ],
+            width: 400,
+            height: 400,
+        };
+        const labelPosition: vec.Vector2 = [500, 500];
+        const expected = [400 + fontSize * 1.25, 400 + fontSize * 1.25];
+
+        expect(clampLabelPosition(labelPosition, graphInfo)).toEqual(expected);
+    });
+    it("should clamp the label position up to the min values", () => {
+        const graphInfo: GraphDimensions = {
+            range: [
+                [-10, 10],
+                [-10, 10],
+            ],
+            width: 400,
+            height: 400,
+        };
+        const labelPosition: vec.Vector2 = [-500, -500];
+        const expected = [-fontSize * 1.5, -fontSize * 2];
+
+        expect(clampLabelPosition(labelPosition, graphInfo)).toEqual(expected);
+    });
```

Comment:
Could you add a test for the case where no explicit clamping was necessary? Like where it would use the `labelPosition[X]` and `labelPosition[Y]` values?